### ISSUE TITLE
let's make infobox harder to open

### ISSUE THREAD

**Issue by eroux** [buda-base/public-digital-library#961]
Currently when clicking on any information, the info box opens, which I would say is the most frequently annoyance on the website. For instance on https://library.bdrc.io/show/bdr:P6161 if we click anywhere on the text of a note we get the box opening, and selecting text from the note to copy/paste is actually pretty hard. Can we make the info box open only when the user clicks on the "i" on the right?

Also, not exactly the same issue but related: at the end of the note the lang tag is present (in small prints). This is really an artefact of the initial optimism about Linked Data. Let's remove it.

**Comment by roopeux** [comment]
I agree with Élie about copying text and the info boxes.  This is quite important.

In general, let's pay attention to copying text throughout the site, because our users probably do that a lot.

**Comment by berger-n** [comment]
done, it should be better now: https://library-dev.bdrc.io/show/bdr:MW25983_DC753D

![simplescreenrecorder-2024-11-22_18 24 38 mkv](https://github.com/user-attachments/assets/f64bb525-96ac-4c4c-b299-1534e7393ca3)

**Comment by roopeux** [comment]
It got much better, but since this is a pretty important detail, can we try to make it behave completely normally?  See the video with sound.

https://github.com/user-attachments/assets/7b224a78-9738-4f5d-aeb3-7c4237067ca3

**Comment by berger-n** [comment]
text copy behavior should be fixed now: https://library-dev.bdrc.io/show/bdr:MW25983_DC753D

![simplescreenrecorder-2024-11-26_11 48 37 mkv](https://github.com/user-attachments/assets/884717bc-8de9-4e68-bd05-90184761b6d6)

**Comment by eroux** [comment]
very nice, thanks! that makes a big difference
